droidlet/tools/crowdsourcing/sync_whitelists.py

```diff
@@ -102,8 +113,37 @@ def add_workers_to_quals(add_list: list, qual: str):
         # Check to make sure the qualification was added successfully
         if not worker.is_qualified(qual):
             logging.info(f"!!! {worker} not successfully qualified, debug")
+            add_all_success = False
         else:
             logging.info(f"Worker {worker.worker_name} added to list {qual}")
+    return add_all_success
+
+
+def revoke_worker_qual(turker: str, qual: str):
```

Comment:
Is this function still needed, if we are never taking workers off any of the qual lists?